proto/src/v1.rs

```diff
@@ -235,6 +235,7 @@ pub enum OperationError {
     NoMatchingAttributes,
     CorruptedEntry(u64),
     CorruptedIndex(String),
+    // TODO: this should just be a vec of the ConsistencyErrors, surely?
```

Comment:
Yeah it should. I had some past ideas here but forgot them. Likely consistency error needs to wrap operr and no the other way for ? To work too.

Reply:
We should be able to something like a `filter_map` so it's just a `Vec` without the `Result` bit... will deal with it later.